src/components/Card/styles.jsx

```diff
@@ -0,0 +1,169 @@
+import styled, { css } from 'styled-components';
+
+const borderRadius = css`
+  border-radius: 0.5rem;
+`;
+
+export const StyledContainer = styled.div`
+  position: absolute;
+  transform-style: preserve-3d;
+  top: -7.5rem;
+  transition: transform 1s;
+  background-image: linear-gradient(to top left, #ff512f, #f09819);
+  color: ${({ theme }) => theme.colors.main};
+  box-shadow: ${({ theme }) => theme.effects.boxShadow};
+  ${borderRadius};
+`;
+
+export const StyledFront = styled.div`
+  display: grid;
+  grid-template-rows: auto;
+  backface-visibility: hidden;
+  width: 28rem;
+  height: 16rem;
+  ${borderRadius};
+  overflow: hidden;
+
+  header {
```

Comment:
Please, read it carefully, use &  for such cases to avoid any issues with selectors specificity
https://webdelving.com/blog/styled-components-ampersand/

Reply:
It is also relevant for all other cases across your project